www/views/page/content/create.php

```diff
@@ -85,6 +81,11 @@ function printButtons($chars): void
 ?>
     </div>
 
+    <div class='form-group'>
```

Comment:
Voeg even wat ruimte toe aan de bovenkant hier, de knopjes voor speciale tekens zitten hier nu heel dicht op in vergelijking met hoe de rest van de onderdelen gepositioneerd zijn.